testdata/test_replacer.py

```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+
+if len(sys.argv) == 1:
+    directory = os.getcwd()
+else:
+    directory = sys.argv[1]
+
+for filename in os.listdir(directory):
```

Comment:
Don't submit this script. I want people to actually look at the images not just blindly replace them.

... I guess I could be convinced given that github has a decent comparison view. But even if it was kept this would go in the `dev/` subdirectory not `testdata/`.

Reply:
lol I didn't mean to stage this, my bad. 
I did make it because inspecting staged changes is easier in my editor and the github comparison view is better than looking at both images when I'm expecting them not to change.